core/vm/interface.go

```diff
@@ -57,6 +57,15 @@ type StateDB interface {
 	// is defined according to EIP161 (balance = nonce = code = 0).
 	Empty(common.Address) bool
 
+	AddrInAccessList(addr common.Address) bool
+	SlotInAccessList(addr common.Address, slot common.Hash) (addressOk bool, slotOk bool)
+	// AddAccessListAccount adds the given address to the access list. This operation is safe to perform
+	// even if the feature/fork is not active yet
+	AddAccessListAccount(addr common.Address)
```

Comment:
Nit, the setter is `Account`, but the getter is `Addr`/address. Pls unify the two, either is fine.

Perhaps also make it a bit easier on the eye? `AddAccessListAccount` -> `AddAddrToAccessList` (to keep with the `AddInAccessList` style.